cmd/kubeadm/app/cmd/reset.go

```diff
@@ -213,10 +215,8 @@ func resetWithCrictl(execer utilsexec.Interface, dockerCheck preflight.Checker,
 
 // cleanDir removes everything in a directory, but not the directory itself
 func cleanDir(filePath string) error {
-	// If the directory doesn't even exist there's nothing to do, and we do
-	// not consider this an error
 	if _, err := os.Stat(filePath); os.IsNotExist(err) {
-		return nil
+		return err
```

Comment:
Do not change here.
This should be `nil`, since we do not consider such errors for non-existent files or dirs. No need to clean/remove empty dir.

Reply:
The caller only print the error when `cleanDir` returns error: `fmt.Printf("[reset] Failed to remove directory: %q [%v]\n", dir, err)`, so I'm on the fence here. I'd like to know other folks' thoughts.

Reply:
it essentially turns missing dirs from `cleanDir` into a warning, which could be undesired.
i think one main reason i've added this was for better test coverage and i can't recall anything else as a good reason.

if this commit is controversial and you want it removed let me know.

Reply:
@dixudx @xiangpengzhao updated PR to remove this first commit / change.